.gitignore

```diff
@@ -15,7 +15,6 @@ pnpm-debug.log*
 
 # Editor directories and files
 .idea
-.vscode
```

Comment:
I'd prefer to not include `.vscode` folder. We could add `.vscode/extensions.json` and `.vscode/settings.json` files into git forcely event if we exclude `.vscode` folder.

Reply:
Since there might have other unexpected files in `.vscode`, such as `.vscode/launch.json` and `.vscode/tasks.json`.